src/tests/tribol_enzyme_jacobian.cpp

```diff
@@ -0,0 +1,926 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+//-----------------------------------------------------------------------------
+//
+// file: tribol_enzyme_jacobian.cpp
+//
+//-----------------------------------------------------------------------------
+
+#include <iostream>
+
+#include "tribol/config.hpp"
+
+#include "gtest/gtest.h"
+
+#ifdef TRIBOL_USE_UMPIRE
+#include "umpire/ResourceManager.hpp"
+#endif
+
+#include "redecomp/common/TypeDefs.hpp"
+
+#include "tribol/physics/Mortar.hpp"
+#include "tribol/interface/tribol.hpp"
+
+namespace tribol {
+
+void FDCheck( double* x1, double* x2, double* n1, double* p1 );
+
+/**
+ * @brief Test fixture for the Enzyme-computed Jacobian terms of the mortar method, not including the nodal normal
+ * contribution.
+ */
+class EnzymeJacobianTest : public testing::Test {
```

Comment:
I'm pendantic here, but to better contrast this class with the "EnzymeMergedJacobianTest" (also see comment there), perhaps we call this "EnzymePartialJacobianTest"? Also, the word "approximate" is sort of overloaded. While this is an approximate Jacobian since not all contributions are in here, you also compute approximate terms using finite difference.

Reply:
Changed to `EnzymeMortarJacobianTest` and to `SimplifiedJacobianCheck`